docs/source/admin/traffic_ops/using.rst

```diff
@@ -293,135 +293,136 @@ The fields in the Delivery Service view are:
+|                                       | - 0 Do not cache (ranges requested from files that are already cached due to a non range request will be a HIT)                                                                                                   |
+|                                       | - 1 Use the `background_fetch <https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/background_fetch.en.html>`_ plugin.                                                                            |
+|                                       | - 2 Use the cache_range_requests plugin.                                                                                                                                                                          |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Raw Remap Text                        | For HTTP and DNS deliveryservices, this will get added to the end of the remap line on the cache verbatim. For ANY_MAP deliveryservices this is the remap line. [1]_                                              |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Regex Remap Expression                | Regex Remap rule to apply to this delivery service at the Edge tier. See `ATS documentation on regex_remap <https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/regex_remap.en.html>`_. [1]_      |
+|                                       |                                                                                                                                                                                                                   |
+|                                       | **Note:** you will not be able to save a Regex Remap Expression if you have Query String Handling set to drop query strings at the edge. See :ref:`rl-regex-remap`.                                               |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Regional Geo-Blocking                 |                                                                                                                                                                                                                   |
```

Comment:
Can you please add a link to `docs/source/admin/quick_howto/regionalgeo.rst` here?

Reply:
done